src/aclic.adoc

```diff
@@ -167,20 +167,30 @@ https://creativecommons.org/licenses/by/4.0/.
 == Introduction
 
 The set of extensions defined in this specification enhance the existing interrupt specifications.
-The Advanced Core Local Interrupt Controller (ACLIC) is a single hart interrupt controller defined for:
+
+The Advanced Interrupt Architecture (AIA) defines two types of external interrupt controller:
+
+  * An interrupt controller that closely coupled with a hart (IMSIC, supporting only message-signaled interrupts).
+  * A shared, memory-mapped, platform interrupt controller for wired platform interrupts (APLIC)
+
+This specification introduces a third type of AIA compliant interrupt controller. 
+
+The Advanced Core Local Interrupt Controller (ACLIC) is a single hart external interrupt controller optimized for real-time systems:
 
   * Low latency interrupt handling (achieved through nested preemption)
   * High throughput interrupt handling
   * Up to 1023 external interrupts
   * Low area overhead
-  * Maximize compatibility with AIA

```

Comment:
why was this removed? Note that this still exists as bullet for Smaclic, as subgoals of the main goals